content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -92,11 +92,12 @@ __egress__: Each `NetworkPolicy` may include a list of whitelist `egress` rules.
 So, the example NetworkPolicy:
 
 1. isolates "role=db" pods in the "default" namespace for both ingress and egress traffic (if they weren't already isolated)
-2. allows connections to TCP port 6379 of "role=db" pods in the "default" namespace from:
+2. (ingres rules) allows connections to all pods in the “default” namespace with the label “role=db” on TCP port 6379 from:
```

Comment:
"ingress" and "egress" both end with a double "s". (But other than that, the edits are factually correct now.)